### PR TITLE
add proper service agent reporting

### PR DESCRIPTION
**Pull request by tangiel**

### CODE REVIEW

**Comment by lookuptable** [comment]
Build is failing. Are you working on a fix?

**Comment by tangiel** [comment]
Yes, though this is a bizarre error.

**Comment by tangiel** [comment]
Tests should be fixed now.

**Comment by tangiel** [comment]
Can you take a look @lookuptable?

File: endpoints-control/src/main/java/com/google/api/control/model/OperationInfo.java [hunk 1]
```diff
@@ -20,12 +20,20 @@
...
+import com.google.common.collect.Maps;
 import com.google.protobuf.Timestamp;
 
+import java.util.Map;
+
 /**
  * Holds basic information about an operation to be obtained from the HTTP layer.
  */
 public class OperationInfo {
+  public static final String SCC_CALLER_IP = "servicecontrol.googleapis.com/caller_ip";
```
Comment (lookuptable) [inline endpoints-control/src/main/java/com/google/api/control/model/OperationInfo.java]: Might make sense to assign the common prefix "servicecontrol.googleapis.com/" to a constant, but it's not a big deal

**Pull request merged**